packages/ui/src/components/ChannelSearch/SearchStatus.tsx

```diff
@@ -0,0 +1,85 @@
+import { makePrettyShortDate } from '@tloncorp/shared/src/logic/utils';
+import { ColorTokens } from 'tamagui';
+
+import { SizableText, View, XStack, YStack } from '../../core';
+import { LoadingSpinner } from '../LoadingSpinner';
+import { SearchState } from './types';
+
+export function SearchStatus({
+  search,
+  color,
+}: {
+  search: SearchState;
+  color?: ColorTokens;
+}) {
+  const { searchComplete, errored, numResults, searchedThroughDate } = search;
+
+  return (
+    <XStack justifyContent="center" alignItems="center">
+      {errored ? (
+        <SizableText size="$s" color="$negativeActionText">
```

Comment:
We should consider breaking some of this into other components -- eg `SmallErrorMessage` or w/e. Might not be quite there yet, but soon.